melange.yaml

```diff
@@ -29,6 +29,9 @@ pipeline:
 
   - runs: |
       cd melange
+      # Mitigate GHSA-m425-mq94-257g
+      go get google.golang.org/grpc@v1.58.3
```

Comment:
Looks like this is already bumped in main but not v0.5.1. Wouldn't we just cut a new melange release or minor release? Or is this the normal process?

Reply:
Generally, I'd say we could cut a release. But I am not sure what is our cadence about it. Let me ask the team.